app/controllers/permissions.js

```diff
@@ -99,8 +99,10 @@ function contacts(e) {
 
 		// We have to actually use a Ti.Contacts method for the permissions to be generated
 		// FIXME: https://jira.appcelerator.org/browse/TIMOB-19933
-		log.args('Ti.Contacts.getAllGroups', Ti.Contacts.getAllGroups());
-
+		if(OS_IOS){
+			log.args('Ti.Contacts.getAllGroups', Ti.Contacts.getAllGroups());
```

Comment:
Ah, didn't see this was iOS only. But the thing is, with this in `OS_IOS` it won't add the required `uses-permission` to `tiapp.xml` anymore for Android. So I'll replace it with `Ti.Contacts.getAllPeople(1)` after merge.